packages/pyxis-react/src/stories/components/Autocomplete/Overview.tsx

```diff
@@ -181,7 +179,16 @@ const accessibilityDescription = (
 
 const classDescription = (
   <p>
-    Autocomplete is based upon this list of CSS classes.
+    The list of Texts CSS classes is available&nbsp;
+    <a
+      className="link"
+      href="https://prima.design/3794e337c/p/26b9e3-autocomplete"
```

Comment:
we generally use the direct link to the 'code' tab
```suggestion
      href="https://prima.design/3794e337c/p/26b9e3-autocomplete/b/54df21"
```